samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -358,17 +365,26 @@ void main(void)
 
 		create_sync();
 
+		sync_wait = true;
+		sync_terminated = false;
+
 		printk("Waiting for periodic sync...\n");
 		err = k_sem_take(&sem_per_sync, K_MSEC(sync_create_timeout_ms));
-		if (err != 0) {
-			printk("failed (err %d)\n", err);
+		if (err != 0 || sync_terminated) {
+			if (err != 0) {
+				printk("failed (err %d)\n", err);
+			} else {
+				printk("terminated\n");
+			}
```

Comment:
```suggestion
			}
			
```

Reply:
Corrected

Reply:
Did you forget to push this change? :)

Reply:
There is added empty line below.

Reply:
Ah, sorry, was expecting GH to mark this as "updated" :)